### PR TITLE
tutorial docs updates

### PR DESCRIPTION
**Pull request by cyrush**

### CODE REVIEW

**Comment by cyrush** [comment]
resolves #1313

**Comment by cyrush** [comment]
@nicolemarsaglia  a few docs update please give them a once over :-)

**Comment by cyrush** [comment]
https://ascent--1339.org.readthedocs.build/en/1339/

File: src/docs/sphinx/BuildingAscent.rst [hunk 1]
```diff
@@ -13,9 +13,8 @@ This page provides details on several ways to build Ascent from source.
 
 For the shortest path from zero to Ascent, see :doc:`QuickStart`.
 
-To build third party dependencies we recommend using :ref:`uberenv <building_with_uberenv>` which leverages Spack or :ref:`Spack directly<building_with_spack>`.
-We also provide info about :ref:`building for known HPC clusters using uberenv <building_known_hpc>`.
-and a :ref:`Docker example <building_with_docker>` that leverages Spack.
+To build third party dependencies we recommend using :ref:`build_ascent.sh`, :ref:`uberenv <building_with_uberenv>` which leverages Spack, or :ref:`Spack directly<building_with_spack>`. We also provide a :ref:`Docker example <building_with_docker>` that leverages Spack.
```
Comment (nicolemarsaglia) [inline src/docs/sphinx/BuildingAscent.rst]: ref for the build_ascent.sh is not working. Do we need < > in there?

File: src/docs/sphinx/BuildingAscent.rst [hunk 2]
```diff
@@ -13,9 +13,8 @@ This page provides details on several ways to build Ascent from source.
...
+.. info about :ref:`building for known HPC clusters using uberenv <building_known_hpc>`. and
```
Comment (nicolemarsaglia) [inline src/docs/sphinx/BuildingAscent.rst]: I think this is an unfinished thought? broken reference and I think also wants to point at build_ascent.sh

Comment (cyrush) [reply src/docs/sphinx/BuildingAscent.rst]: last part os old context I moved and commented out, I'll re-review as rendered.

Comment (nicolemarsaglia) [reply src/docs/sphinx/BuildingAscent.rst]: Ah gotcha, weird that it rendered for me. 
bad_docs_render

Comment (cyrush) [reply src/docs/sphinx/BuildingAscent.rst]: looks like i botched the spacing for the comment,thanks!

File: src/docs/sphinx/BuildingAscent.rst [hunk 3]
```diff
@@ -471,20 +486,14 @@ Building Ascent in a Docker Container
...
 
-.. literalinclude:: ../../../scripts/build_ascent/build_ascent.sh
-   :language: bash
+Special Case Builds
+------------------------
 
-This script supports building on linux, macOS, and windows.
-It supports building without device support, and with CUDA or ROCm/HIP.
+BabelFlow Support
+~~~~~~~~~~~~~~~~~
```
Comment (nicolemarsaglia) [inline src/docs/sphinx/BuildingAscent.rst]: for some reason this title is not being rendered for me. 
no_babelflow

File: src/docs/sphinx/BuildingAscent.rst [hunk 4]
```diff
@@ -595,7 +604,7 @@ Here is script that shows how to build additional dependencies for bflow-stats (
 .. _paraview_ascent_support:
 
 ParaView Support
-----------------
+~~~~~~~~~~~~~~~~~
```
Comment (nicolemarsaglia) [inline src/docs/sphinx/BuildingAscent.rst]: Missing this title as well
no_paraview_title

**Review by nicolemarsaglia — APPROVED** [review]
Looks good!

**Comment by cyrush** [comment]
thanks @nicolemarsaglia !

**Comment by cyrush** [comment]
visited by the occasional data binning failure

**Pull request merged**